assets/data-port/export/export-job-state.js

```diff
@@ -0,0 +1,186 @@
+import apiFetch from '@wordpress/api-fetch';
+
+/**
+ * @typedef Job
+ *
+ * @property {Object}   status            Job status.
+ * @property {string}   status.status     Job status name.
+ * @property {number}   status.percentage Job progress percentage.
+ * @property {string}   id                Job ID.
+ * @property {boolean}  deleted           Was the job deleted.
+ * @property {Object}   result            Job result.
+ * @property {Object[]} files             Job files.
+ * @property {Object}   error             Error message.
+ */
+
+/**
+ * Client for the export API.
+ *
+ * @param {Function} updateState
+ * @return {{cancel: Function, start: Function, reset: Function, update: Function}} Export store actions.
+ */
+export function ExportJobState( updateState ) {
```

Comment:
I tried out working with a simpler pattern for this instead a full data store since it's a pretty small scope for that.

Reply:
I'm wondering if the reducer pattern would be better for this case. If we don't want to save the data in the store, maybe we could just use the `useReducer` with a reducer and the actions. I think the advantages would be to use a more common pattern for React (maybe more natural), and if in some time we decide save it in the store for some reason, it also would be easier to refactor. WDYT?